src/gui/widgets/TextFloat.cpp

```diff
@@ -124,6 +125,8 @@ TextFloat * TextFloat::displayMessage(const QString & title,
 		QTimer::singleShot(timeout, tf, SLOT(close()));
 	}
 
+	QCoreApplication::processEvents();
```

Comment:
IMO this is the wrong place to fix the problem. Instead the problem(s) should be fixed in the places where long running operations are performed in the GUI main thread, e.g. in the VST loading code, etc.

The `TextFloat` as is works perfectly okay and you would see similar problems with other widgets if they are about to be shown before something that blocks the GUI main thread.